customer-service-tests/EmployeeFactoryTests.cs

```diff
@@ -14,15 +14,35 @@ public class EmployeeFactoryTests
         [InlineData(3)]
         public void CanGetEmployeeById(int id)
         {
-            //CustomerFactory customerFactory = new CustomerFactory();
-            //var employee = customerFactory.get(id);
+            EmployeeFactory employeeFactory = new EmployeeFactory();
+            var employee = employeeFactory.get(id);
 
-            //var expectedType = typeof(Employee);
-            //var actualType = typeof(employee);
+            Assert.NotNull(employee);
+            Assert.True(employee.GetType() == typeof(Employee));
+            Assert.True(employee.FirstName.Length > 0);
+            Assert.True(employee.LastName.Length > 0);
+            Assert.True(employee.DepartmentId.GetType() == typeof(int));
+            Assert.True(employee.Administrator.GetType() == typeof(bool));
```

Comment:
`Assert.IsType` is a few lines above this `foreach` loop. These are here to ensure that no property comes back as `null` on any Employee record.